packages/react/src/auto/polaris/PolarisAutoForm.tsx

```diff
@@ -90,6 +90,9 @@ const PolarisAutoFormComponent = <
 
   const formContent = props.children ?? (
     <>
+      <Text variant="headingLg" as="h5">
+        {humanizeCamelCase(action.operationName)}
```

Comment:
Do we need to memoise the function result with `useMemo`?

Reply:
Good call. Updated to memoize these humanized names